backup.go

```diff
@@ -34,20 +34,28 @@ import (
 // than the maxBatchSize.
 const flushThreshold = 100 << 20
 
-// Backup is a wrapper function over Stream.Backup to generate full and incremental backups of the
-// DB. For more control over how many goroutines are used to generate the backup, or if you wish to
-// backup only a certain range of keys, use Stream.Backup directly.
+// Backup dumps a protobuf-encoded list of all entries in the database into the
+// given writer, that are newer than or equal to the specified version. It
+// returns a timestamp (version) indicating the version of last entry that is
+// dumped, which after incrementing by 1 can be passed into later invocation to
+// generate incremental backup, of entries that have been added/modified since
```

Comment:
remove the comma.

Reply:
Done. Added comment in related issue on Github too.